system-int-test/test/cmr/system_int_test/ingest/provider_ingest_test.clj

```diff
@@ -125,10 +127,19 @@
           gran4 (d/ingest "PROV2" (dg/granule-with-umm-spec-collection coll3 "C1-PROV1"))
           variable1 (variables/ingest-variable-with-attrs {:native-id "var1"
                                                            :Name "Variable1"})
-
           variable2 (variables/ingest-variable-with-attrs {:native-id "var2"
                                                            :Name "Variable2"
                                                            :provider-id "PROV2"})
+          service1 (services/ingest-service-with-attrs {:native-id "svc1"
+                                                        :Name "service1"})
+          service2 (services/ingest-service-with-attrs {:native-id "svc2"
+                                                        :Name "service2"
+                                                        :provider-id "PROV2"})
+          _ (index/wait-until-indexed)
+          sa1 (au/make-service-association
```

Comment:
Is there a more descriptive name that we can give `sa1` and `sa2`? Something like `associate-s1-to-coll1` or whatever that case may be. It would make understanding and modifying tests much easier in the future

Reply:
Renamed `sa1` to `svc-association1`.